test_crates/enum_repr_int_changed/old/src/lib.rs

```diff
@@ -22,30 +22,85 @@ pub enum IsizeToUsizeEnum {
     Baz,
 }
 
-// The following enums have *removals* of repr(i*) and repr(u*),
-// not changes to another repr(i*) or repr(u*).
-// They should not be reported by this rule, because they have their own rule.
+#[repr(u8, C)]
```

Comment:
Can you throw in test cases for `u64 -> usize` and `i64 -> isize`, just to be safe? `usize/isize` on 64bit x86 are representationally identical to `u64/i64`, but this is still a breaking change (because there's no guarantee that 64bit x86 is the only architecture where a given crate will run) and it would be great to be 100% sure that we catch and report it as such.

Reply:
These are great life-based test cases, I added them!